mne_bids/dig.py

```diff
@@ -0,0 +1,427 @@
+"""Read/write BIDS compatible electrode/coords structures from MNE."""
+import json
+from collections import OrderedDict
+
+import numpy as np
+from mne.channels.montage import make_dig_montage
+from mne.io.constants import FIFF
+from mne.utils import _check_ch_locs, logger, warn
+
+from mne_bids.config import (COORD_FRAME_DESCRIPTIONS,
+                             BIDS_IEEG_COORDINATE_FRAMES,
+                             BIDS_MEG_COORDINATE_FRAMES,
+                             BIDS_EEG_COORDINATE_FRAMES,
+                             BIDS_COORDINATE_UNITS,
+                             MNE_TO_BIDS_FRAMES, BIDS_TO_MNE_FRAMES,
+                             _frame_to_str)
+from mne_bids.tsv_handler import _from_tsv
+from mne_bids.utils import (_extract_landmarks, _parse_bids_filename,
+                            _scale_coord_to_meters, _write_json, _write_tsv,
+                            make_bids_basename
+                            )
+
+
+def _electrodes_tsv(raw, fname, kind, overwrite=False, verbose=True):
+    """
+    Create an electrodes.tsv file and save it.
+
+    Parameters
+    ----------
+    raw : instance of Raw
+        The data as MNE-Python Raw object.
+    fname : str
+        Filename to save the electrodes.tsv to.
+    kind : str
+        Acquisition type to save electrodes in. For iEEG, requires size.
```

Comment:
humm ... where will get this from?

Reply:
also, "kind" is often rather described as the "data modality" (eeg, ieeg, meg, ...) ... the word "acquisition" is too reminiscent of the `acq` _entity_ which does a different thing than the `kind` _suffix_